frontend/src/pages/ChatPage.vue

```diff
@@ -1,52 +1,52 @@
 <template>
   <v-container>
-    <h1>Chat Page, Room Title</h1>
+    <h1>{{ roomInfo.title }}</h1>
     <v-row>
       <v-col cols="12">
-        <li v-for="message in messages" :key="message.title">
+        <li v-for="message in messages" :key="message.title" class="pb-1">
```

Comment:
[MUST]
```suggestion
        <li v-for="message in messages" :key="message.uuid" class="pb-1">
```

Reply:
[Q]
whats `pb-1`?

Reply:
The `pb-1` is **p**adding **b**ottom - 4px . See https://vuetifyjs.com/ja/styles/spacing/ .

Reply:
did you fix this?
https://github.com/tmpchat/tmpchat/pull/39#discussion_r483149456

Reply:
Oops. Fixed in b37d051a .